sphinxcontrib/chapeldomain/__init__.py

```diff
@@ -35,21 +35,21 @@
 
 from sphinxcontrib.chapeldomain.chapel import ChapelLexer
 
-VERSION = '0.0.23'
+VERSION = '0.0.24'
```

Comment:
This shouldn't be updated.  I've taken the tactic of updating the release number immediately after each new release is pushed.  The most recent release was `0.0.22`

Reply:
oh, gotcha. I misunderstood and I had used this to make sure I was getting the right build while testing. I'll revert it to `.23` then?

Reply:
Yeah, that'd be good.  Thanks!